modules/channel-teams/src/backend/client.ts

```diff
@@ -141,20 +141,28 @@ If you have a restricted app, you may need to specify the tenantId also.`
   }
 
   private _sendIncomingEvent = async (activity: Activity, threadId: string) => {
-    const { text, from, type } = activity
-
-    await this.bp.events.sendEvent(
-      this.bp.IO.Event({
-        botId: this.botId,
-        channel: 'teams',
-        direction: 'incoming',
-        payload: { text },
-        preview: text,
-        threadId,
-        target: from.id,
-        type
```

Comment:
There are two behaviors here that are not taken over with the port to messaging
- `type: 'text` wasn't added to `payload`. Now it is
- The `type` property of the event was taken from teams directly. Now we just put 'text'